Sources/Shared/UI/Social/ActivityPictureTableViewCell.m

```diff
@@ -119,12 +119,12 @@ - (void)setSocialActivityStreamForSpecificContent:(SocialActivity *)socialActivi
                     }
                 }
                 
-                self.activityMessage.attributedText = attributedMessage;
+                self.lbFileName.attributedText = attributedMessage;
             } else {
-                self.activityMessage.text = @"";
+                self.lbFileName.text = @"";
             }
             
-            _lbFileName.text = @"";
+            self.activityMessage.attributedText = socialActivityStream.attributedMessage;
             if ([socialActivityStream.templateParams valueForKey:@"mimeType"] && [[socialActivityStream.templateParams valueForKey:@"mimeType"] rangeOfString:@"image"].location != NSNotFound) {
```

Comment:
Fix error: before display the name of the photo (or file) at the message label
In fact in picture cell: We have a message label to display the message tapped by user & a FileName label to display the name of the photo.